master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -4,36 +4,14 @@ canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/i
 ---
 
 This document outlines how to install {{site.prodname}} on a cluster initialized with 
-[kubeadm](http://kubernetes.io/docs/getting-started-guides/kubeadm/).  {{site.prodname}}
-is compatible with kubeadm-created clusters, as long as the [requirements](#requirements) are met. 
-
-## Requirements
-
-For {{site.prodname}} to be compatible with your kubeadm-created cluster:
-
-* It must be running at least Kubernetes v1.7
-
-* There should be no other CNI network configurations installed in /etc/cni/net.d (or equivalent directory)
-
-* The kubeadm flag `--pod-network-cidr` must be set when creating the cluster with `kubeadm init` 
-  and the CIDR(s) specified with the flag must match {{site.prodname}}'s IP pools. The default 
-  IP pool configured in {{site.prodname}}'s manifests is `192.168.0.0/16`
-
-* The CIDR specified with the kubeadm flag `--service-cidr` must not overlap with 
-  {{site.prodname}}'s IP pools
-  
-  * The default CIDR for `--service-cidr` is `10.96.0.0/12`
-  
-  * The default IP pool configured in {{site.prodname}}'s manifests is `192.168.0.0/16`
-
-You can create a cluster compatible with these manifests by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+[kubeadm](http://kubernetes.io/docs/getting-started-guides/kubeadm/).  
 
 ## Installing {{site.prodname}} with a Kubernetes-hosted etcd
 
 As a non-production quick start, to install {{site.prodname}} with a single-node dedicated etcd cluster,
 running as a Kubernetes pod:
 
-1. Ensure your cluster meets the [requirements](#requirements) (or recreate it if not).
+1. If you have an existing cluster, ensure that it meets the [requirements](../../../reqs/system). Otherwise, you can create a cluster compatible with these manifests by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
```

Comment:
Yeah, again I think "the requirements" reads a bit awkwardly compared to "these requirements"

Reply:
I think we need a definite article here. We are referring to a specific page, which can be accessed by clicking on the hyperlink. The link text corresponds to the title of that page. However, I've added some more words which hopefully solves for the abrupt feeling that you mention.

Reply:
I don't think it's fixed the feeling... maybe it's just me, but "the requirements" just feels wrong. There are a lot of requirements in the world. Maybe it needs to say "Calico's installation requirements" or something to make it more specific, but I feel like "the requirements" is just too vague or something.